AvansDevOps/ForumComposite/ForumMessageComponent.cs

```diff
@@ -8,12 +9,25 @@ namespace AvansDevOps.ForumComposite
 {
     public class ForumMessageComponent : ForumComponent
     {
-        private string message;
-        public ForumMessageComponent(string message)
+        private ForumCompositeComponent? parent;
+        public string message { get; private set; }
+        public User creator { get; private set; } 
+        public ForumMessageComponent(string message, User creator)
         {
             this.message = message;
+            this.creator = creator;
         }
 
-        public string GetMessage() => message;
+        public void SetParent(ForumCompositeComponent parent) => this.parent = parent;
+
+        public int AddMessage(ForumMessageComponent component)
```

Comment:
Ik weet niet of ik het goed begrijp, maar verdwijnt hierdoor niet het bericht waarop is gereageerd ? Vgm maak je hier een nieuwe thread en voeg je alleen de reactie toe ?

Reply:
nee het bericht waarop is gereageerd wordt meegegeven bij het maken van de thread als "base" message